routes/index.js

```diff
@@ -19,6 +19,7 @@ router.get('/api/logout', userAPI.logout)
 router.post('/api/tutorial/create', tutorialAPI.createTutorial)
 router.post('/api/tutorial/update', tutorialAPI.updateTutorial)
 router.post('/api/tutorial/delete', tutorialAPI.removeTutorial)
+router.post('/api/tutorial', tutorialAPI.getTutorial)
```

Comment:
I would suggest a `GET` for this, such as `GET /api/tutorial/:id`

Reply:
We should probably change the file api endpoints in the future, thanks for pointing out.

For now let's go with `GET` for the tutorials.